lib/graphql/railtie.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require_relative './upgrader/member'
+require_relative './upgrader/schema'
+
+module GraphQL
+  class Railtie < Rails::Railtie
+    rake_tasks do
+      namespace :graphql do
+        task :upgrade, [:dir] do |t, args|
+          unless (dir = args[:dir])
+            fail 'You have to give me a directory where your GraphQL schema and types live. ' \
+             'For example: `bin/rake graphql:upgrade[app/graphql/**/*]`'
```

Comment:
`bin/rake` could be `bin/rails` in Rails 5, right? :sweat:

Reply:
Yeah, but we want to support Rails 4 and even 3 (looking at you, Github)